src/services/listS3Objects.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  S3Client,
+  ListObjectsV2Command,
+  ListObjectsV2CommandInput
+} from '@aws-sdk/client-s3'
+
+export const listS3Objects = async (
+  input: ListObjectsV2CommandInput,
+  objects: string[] = []
+): Promise<string[]> => {
+  const client = new S3Client({ region: 'eu-west-2' })
```

Comment:
Would it make sense for this to be an env var

Reply:
Yes, and actually you can just use `AWS_REGION` because I believe the Lambda runtime just provides it anyway. I was trying to find an official source for this, but check out e.g. this SO answer: https://stackoverflow.com/a/43095295

Reply:
Fixed.